pkg/generate/labels.go

```diff
@@ -41,12 +41,20 @@ func managedBy(labels map[string]string) {
 }
 
 func generatedBy(labels map[string]string, triggerResource unstructured.Unstructured) {
-	key := "kyverno.io/generated-by"
-	value := fmt.Sprintf("%s-%s-%s", triggerResource.GetKind(), triggerResource.GetNamespace(), triggerResource.GetName())
+	keyKind := "kyverno.io/generated-by-kind"
+	keyNamespace := "kyverno.io/generated-by-namespace"
+	keyName := "kyverno.io/generated-by-name"
```

Comment:
Are we using these labels to identify the source resource (in the webhook)?  I don't find they are used anywhere else. If not, can we trim to 63 characters if exceed the length?

Reply:
As discussed, I have trimmed the label to 63 characters if exceed the length.

Reply:
Can we use a separate label for each? And need to trim the size.